backend/controllers/election.js

```diff
@@ -83,4 +83,25 @@ const getInactiveElections = async (req, res, next) => {
   );
 };
 
-export default { createElection, getActiveElections, getInactiveElections };
+const getElectionById = async (req, res, next) => {
```

Comment:
this controller does not implement all of the required points of the task. 
Missing points:
1.  Only creator should see the ballot votes, everyone else should not get this data.
2.  User should see whether they have voted on the election yet.

I suggest you add some new fields to response json, to make it easy to differentiate between these states on the frontend. Something like this: 
```
{
  isOwner: true | false,
  hasVoted: true | false,
}
```